src/services/etherspot.js

```diff
@@ -57,12 +57,20 @@ import type { Asset, Balance } from 'models/Asset';
 import type { EthereumTransaction, TransactionPayload, TransactionResult } from 'models/Transaction';
 import type { EtherspotTransactionEstimate } from 'models/Etherspot';
 
-
 class EtherspotService {
   sdk: EtherspotSdk;
   subscription: ?Subscription;
+  instances: Array<EtherspotSdk> = [];
+  supportedNetworks: Array<NetworkNames> = [
+    NetworkNames.Mainnet,
+    NetworkNames.Kovan,
+    NetworkNames.Bsc,
+    NetworkNames.Matic,
+    NetworkNames.Xdai,
+  ];
```

Comment:
Since were running account contract compution over all of the networks, it would make sens to do something similar to:
```suggestion
  supportedNetworks: Array<NetworkNames> = [
    (isProdEnv() ? NetworkNames.Mainnet : NetworkNames.Kovan),
    NetworkNames.Bsc,
    NetworkNames.Matic,
    NetworkNames.Xdai,
  ] : [
  
  ]
```
Otherwise we would do contract computation for both mainnet and kovan regardles whether there are actually needed.

Reply:
I considered this but was too tired - it's updated! Great suggestion 👍